src/python/pants/backend/python/util_rules/vcs_versioning.py

```diff
@@ -102,15 +105,19 @@ async def generate_python_from_setuptools_scm(
 
     argv = ["--root", str(maybe_git_worktree.git_worktree.worktree), "--config", config_path]
 
-    result = await Get(
-        ProcessResult,
-        VenvPexProcess(
-            setuptools_scm_pex,
-            argv=argv,
-            input_digest=input_digest,
-            description=f"Run setuptools_scm for {request.protocol_target.address.spec}",
-            level=LogLevel.INFO,
-            cache_scope=ProcessCacheScope.PER_SESSION,
+    result = await process_request_to_process_result(
```

Comment:
I'm doing call-by-name everywhere from now on...

Reply:
Does `implicitly` support simultainous args and kwargs? If so, placing `VenvPexProcess` as an arg reduces indentation levels a smidge.

Reply:
It only supports *args. But we can possibly revisit that.